src/main/scala/com.snowplowanalytics.snowplow/scalatracker/Tracker.scala

```diff
@@ -427,6 +430,44 @@ class Tracker(emitters: Seq[TEmitter], namespace: String, appId: String, encodeB
         System.err.println(s"Failed to retrieve context: $message")
         send(payload)
     }
+ 
+  /** 
+   *  Sends a Snowplow Event when error is non fatal.
+   *  
+   *  @param error The throwable
+   */
+  def trackError(error: Throwable): Unit = error match {
+    case NonFatal(e) =>
+      val stackElement = headStackTrace(e)
+
+      val data =
+        ("message"               -> truncateString(e.getMessage, MaxMessageLength).getOrElse("Null or empty message found")) ~
+          ("stackTrace"          -> truncateString(stackTraceToString(e), MaxStackLength)) ~
+          ("threadName"          -> truncateString(Thread.currentThread.getName, MaxThreadNameLength)) ~
+          ("threadId"            -> Thread.currentThread.getId.toString) ~
+          ("programmingLanguage" -> "SCALA") ~
+          ("lineNumber"          -> stackElement.map(_.getLineNumber.toString)) ~
+          ("className"           -> stackElement.map(_.getClassName).flatMap(truncateString(_, MaxClassNameLength))) ~
+          ("exceptionName"       -> truncateString(e.getClass.getName, MaxExceptionNameLength)) ~
+          ("isFatal"             -> true.toString)
```

Comment:
This will be string `"true"` instead of boolean `true`. It won't pass the validation against [schema](https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow/application_error/jsonschema/1-0-1#L72).